src/internal.c

```diff
@@ -20562,6 +21118,17 @@ int ReceiveData(WOLFSSL* ssl, byte* output, int sz, int peek)
             }
             return ssl->error;
         }
+
+#ifdef WOLFSSL_DTLS13
+        if (ssl->options.dtls) {
+            /* TODO: Dtls13DoScheduledWork(ssl) may return WANT_WRITE, but
```

Comment:
Is this TODO still valid? Can we fix this TODO?

Reply:
Looking better at it, `ReceiveData` may already return `WANT_WRITE` from other code paths (like in renegotiation). I dropped the TODO and the "not supported" part from the comment.